All EIPIP Meetings/Meeting 107.md

```diff
@@ -0,0 +1,239 @@
+# EIPIP Meeting 107
+**Meeting Date/Time: July 17, 2024 at 14:00 UTC**
+
+**Meeting Duration: 34 mins**
+
+**[Github Agenda](https://github.com/ethcatherders/EIPIP/issues/346)**
+
+**[Audio/Video of the meeting:](https://www.youtube.com/watch?v=DgdzaxHWFcA)**
+
+**Moderator: Pooja Ranjan**
+**Notes: Aliyu Adeniji**
+
+### Summary
+| S No | Agenda | Summary |
+| -------- | -------- | -------- |
+| 107.1    |  Pull requests number 1363, and PR number 403|They were already approved earlier but the author made some changes, so clarification is needed from the author before merging them.  |
+| 107.2    | Pull Request on ERC 342, which is updating ERC 2098    | An input was received one from RickMoore, and the PR seems explanatory, and it can be merged without further delay    |
+|  107.3   | There is an update to 1175 and they have pinged a bunch of ER and ERCs    | The PR is concluded and it will be merged     |
+| 107.4     | update EIP 4788 PR number 8590    | there needs to be a further deliberation on the pull request as there is not consensus on the PR yet    |
+| 107.5     | EIP 3712 multiple fungible token    | it is probably in the wrong repo, and it should be moved to the right repository     |
+| 107.6     |  issue 345 on EIPIP GitHub repository    | This is to mark that EIP 7212 has moved because this proposal has moved into RIP repository and now we have RIP 7212, it is resolved and can be marked as resolved    |
+| 107.7    | Report on wallet call API   | The deadline for wallet API interface proposal 5792 is july 24, all thoughts and comments should be dded to the Fellowship of ethereum magician page    |
+| 107.8 | Change in meeting time | It was agreed that the EIPIP meeting should henceforth hold once in every month |
+
+## 1. Discuss Open PRs and proposals
+
+### Final decisions on some isssues and proposals taken
+
+
+**Pooja** [0:00:](https://youtu.be/DgdzaxHWFcA?t=0) Welcome to EIPIP meeting 107 this is issue number 346 on ethcatherders EIPIP
+GitHub repository, on agenda we have some open issues and pull requests to be
+discussed, we'll take a look at some miscellaneous issues, as well as PRs, and we'll collect editor feedback, thoughts, whether we should keep it or close it,  we'll take a look at call for inputs added here and then discussions from earlier meetings I see there is a request for discussion that happened in the last meeting, we can hear about that and we'll get some updates on working group discussion that happened in the middle of the week. So starting from the very first item we do have a list of pull requests the first one is 1363 it seems that it has been already approved earlier but the author has added some new changes, I wonder what the editors think, is it good to be merged or do we want to get some more clarification from the author.
```

Comment:
```suggestion
discussed, we'll take a look at some miscellaneous issues, as well as PRs, and we'll collect editor feedback, thoughts, whether we should keep it or close it,  we'll take a look at call for inputs added here and then discussions from earlier meetings I see there is a request for discussion that happened in the last meeting, we can hear about that and we'll get some updates on working group discussion that happened in the middle of the week. So starting from the very first item we do have a list of pull requests the first one is 1363 it seems that it has been already approved earlier but the author has added some new changes, I wonder what the editors think, is it good to be merged or do we want to get some more clarification from the author.
```
```suggestion
discussed, we'll take a look at some miscellaneous issues, as well as PRs, and we'll collect editor feedback, thoughts, whether we should keep it or close it,  we'll take a look at call for inputs added here and then discussions from earlier meetings I see there is a request for discussion that happened in the last meeting, we can hear about that and we'll get some updates on working group discussion that happened in the middle of the week. 
#### https://github.com/ethereum/ERCs/pull/403

**Pooja**: So starting from the very first item we do have a list of pull requests the first one is 1363 it seems that it has been already approved earlier but the author has added some new changes, I wonder what the editors think, is it good to be merged or do we want to get some more clarification from the author.
```

Reply:
Can you please update the notes to reflect the beginning of discussions as suggested in Row 33?